.github/workflows/build-docker-images.yml

```diff
@@ -307,3 +307,41 @@ jobs:
   #           REF=main
   #         push: true
   #         tags: huggingface/transformers-pytorch-deepspeed-amd-gpu-push-ci
+
+  latest-quantization-torch-docker:
+    name: "Latest Pytorch + Quantization [dev]"
+     # Push CI doesn't need this image
+    if: inputs.image_postfix != '-push-ci'
```

Comment:
Just want to know if we intent to run `quantization` tests in a daily basis, or if we prefer to run them on each commit merged into `main`?

Reply:
I think that it is better to run them on a daily basis. There are many slows tests too + I think that the quantization tests are not easily broken by changes in transformers. The bigger issue is the third party libraries that might introduce breaking changes. cc @younesbelkada

Reply:
Yes daily basis sounds great!

Reply:
OK